service/provider/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -0,0 +1,122 @@
+package gov.va.vro.service.provider.mas.service.mapper;
+
+import gov.va.vro.model.*;
+import gov.va.vro.service.provider.mas.model.MasAnnotType;
+import gov.va.vro.service.provider.mas.model.MasAnnotation;
+import gov.va.vro.service.provider.mas.model.MasCollectionAnnotation;
+import gov.va.vro.service.provider.mas.model.MasDocument;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class MasCollectionAnnotsResults {
+
+  private static final String BP_CONDITION = "Hypertension";
+  private static final String ASTHMA_CONDITION = "Asthma";
+  private static final String BP_SYSTOLIC_CODE = "8480-6";
+  private static final String BP_SYSTOLIC_DISPLAY = "Systolic blood pressure";
+  private static final String BP_DIASTOLIC_CODE = "8462-4";
+  private static final String BP_DIASTOLIC_DISPLAY = "Diastolic blood pressure";
+  private static final String BP_UNIT = "mm[Hg]";
+  private static final String BP_READING_REGEX = "^\\d{1,3}\\/\\d{1,3}$";
+
+  public AbdEvidence mapAnnotsToEvidence(MasCollectionAnnotation masCollectionAnnotation) {
+
+    AbdEvidence abdEvidence = new AbdEvidence();
+
+    List<AbdMedication> medications = new ArrayList<>();
+    List<AbdCondition> conditions = new ArrayList<>();
+    List<AbdProcedure> procedures = new ArrayList<>();
+    List<AbdBloodPressure> bpReadings = new ArrayList<>();
+    boolean isConditionBP = false;
+    boolean isConditionAsthma = false;
+
+    for (MasDocument masDocument : masCollectionAnnotation.getDocuments()) {
```

Comment:
I'm not sure whether this mapping is consistent with Lighthouse FHIR data mapping. But since we don't have the real data to test, this is as good as we can go. It is good to have a lot of info logged here.